backend/src/index.ts

```diff
@@ -85,6 +88,14 @@ io.on('connection', (socket: ChatSocket) => {
     );
   });
 
+  socket.on('reaction', (reactionId) => {
+    console.log(`user ${socket.userId} reaction ${reactionId}`);
+    var reactionMap =  reactions[reactionId] || {};
```

Comment:
to samo co wyżej - z constem śmiga (natomiast jeżeli nie const to zmieńmy na let)

Reply:
zmieniłem na consta